.travis.yml

```diff
@@ -7,6 +7,12 @@ install:
   - mvn -N io.takari:maven:wrapper -Dmaven=3.6.0
   - java -version && javac -version && ./mvnw -version -B
 
+before_script:
+  # Check each method has docs and tests, if missing, fail
+  - find vars -type f -name "*.groovy" -exec bash -c '[ -f vars/$(basename {} .groovy).txt ] || echo "Missing vars/$(basename {} .groovy).txt" >> fail.tmp' \;
+  - find vars -type f -name "*.groovy" -exec bash -c '[ -f test/Jenkinsfile-$(basename {} .groovy) ] || echo "Missing test/Jenkinsfile-$(basename {} .groovy)" >> fail.tmp' \;
+  - [ -f fail.tmp ] && cat fail.tmp && exit 1
```

Comment:
Will this exit the shell in the case of missing docs or tests? Might be ok as I'm assuming that Travis will just exit the current run and report a failure.